src/main/java/gtPlusPlus/core/recipe/RECIPES_GREGTECH.java

```diff
@@ -565,12 +565,6 @@ private static void laserEngraverRecipes() {
     }
 
     private static void breweryRecipes() {
-        GT_Values.RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(14))
```

Comment:
I dont see any reason to remove these conversions.
can do it directly without 'enchanting utils' of course

Reply:
of course even without enchanting utils that will still need some check either that the fluid exists or that openblocks is loaded, to avoid a crash in gt++ devenv